README.md

```diff
@@ -38,27 +42,31 @@ To install this package into a workspace already containing SOSS, just clone thi
 
 5. Source a colcon environment in which ROS2 has been built (soss-ros2 uses rclcpp package).
     ```
-    source path/to/ros2/ws/install/local_setup.bash
+    $ source path/to/ros2/ws/install/local_setup.bash
     ```
 
 6. In the workspace folder, execute colcon:
     ```
-    colcon build --packages-up-to soss-dds soss-ros2
+    $ colcon build --packages-up-to soss-dds soss-ros2
     ```
+
 7. Source the current environment
     ```
-    source install/local_setup.bash
+    $ source install/local_setup.bash
     ```
 
 ## Usage
 
 This system handle is mainly used to connect any system with the DDS protocol.
+Exists two main usages of this *SOSS DDS plugin`:
+- Connection through *UDP* to a DDS cloud
+- Used as a tunnel through TCP between two SOSS systems. See [TCP tunnel](#tcp-tunnel) for more information.
 
 ### Configuration
 
-SOSS must be configured with a YAML file, which tells the program everything it needs to know in order to establish the connection between two or more systems that the user wants. 
-For example, if a simple string message wants to be exchanged between two ROS2 systems through a [TCP tunnel](#tcp-tunnel), 
-the configuration files for the two different SOSS instances which that tunnel must have should look as follows:
+SOSS must be configured with a YAML file, which tells the program everything it needs to know
+in order to establish the connection between two or more systems that the user wants.
+An example of YAML configuration file that connect ROS2 to DDS could be the following:
```

Comment:
"an example of A YAML configuration file"

Reply:
Done!